crates/volta-core/src/path/mod.rs

```diff
@@ -44,6 +45,11 @@ pub fn ensure_volta_dirs_exist() -> Fallible<()> {
         ensure_dir_exists(user_toolchain_dir()?)?;
         ensure_dir_exists(tmp_dir()?)?;
         ensure_dir_exists(log_dir()?)?;
+        // also ensure the basic shims exist
```

Comment:
The windows installer actually already creates the "shims" (in actuality they are copies of the `shim.exe` binary) for these in the `Program Files` directory. Additionally, the directory itself is stored in the Registry, so it's not easily available using the tests. Can we wrap this in a block that only applies to unix, at least until we refactor the update process? (which I believe will 

```rust
#[cfg(unix)]
{
    ensure_shim_exists("node")?;
    ensure_shim_exists("yarn")?;
    ensure_shim_exists("npm")?;
    ensure_shim_exists("npx")?;
}
```

Reply:
Yeah, I just pushed some test code that I thought would work, but didn't actually test on windows yet - I'll cfg these out instead